packages/webapp/src/components/InviteUser/index.jsx

```diff
@@ -44,11 +45,19 @@ export default function PureInviteUser({ onInvite, onGoBack, roleOptions = [] })
     { value: 'OTHER', label: t('gender:OTHER') },
     { value: 'PREFER_NOT_TO_SAY', label: t('gender:PREFER_NOT_TO_SAY') },
   ];
+  const languageOptions = [
+    { value: 'en', label: t('PROFILE.ACCOUNT.ENGLISH') },
+    { value: 'es', label: t('PROFILE.ACCOUNT.SPANISH') },
+    { value: 'pt', label: t('PROFILE.ACCOUNT.PORTUGUESE') },
+    { value: 'fr', label: t('PROFILE.ACCOUNT.FRENCH') },
+  ];
 
   const disabled = !isValid || !isDirty;
   const onSubmit = (data) => {
+    console.log(data);
```

Comment:
I think this was added for testing purpose. Seems a bit unnecessary to print a lot of data